### PR TITLE
Reorganize as countertop package

### LINKED ISSUE #85: Reorgnaize repository to be a package
#### Description
We're moving to releasing this as a package but to do that we need to improve the repository a bit:

1. Remove the concept of `components` (this is going to ONLY be the countertop)

We may also want to remove the use of environment variables for ANY form of configuration, but I will treat that as a separate issue.

#### Relevant Resources / Research
None

#### Related Issues
Issue #80

### PR DESCRIPTION
**Pull request by slifty**
## Description
This PR is a significant change: it moves this repository from being a standalone "application" that can be cloned, configured, and run into being a "countertop" package that a TVK implementation will import (alongside the other appliances / components that it wants to use).

This is the culmination of a series of discussions about what the TVK entry point should look like.

It means we no longer have a `src/components` directory, since this is *only* the countertop.  The countertop classes are now in a `src/classes` directory instead.

We expose all of the classes at the package level, so developers will do things like:

```
import { CountertopCoordinator } from `@tvkitchen/countertop`
const countertop = new CountertopCoordinator()
```

Questions for the room:

1. Should `CountertopCoordinator` just be called `Countertop` (very important)
2. Is `src/classes` the right directory (less important) 
3. Is `@tvkitchen/countertop` an appropriate package name (last call to change it!).

## Due Diligence Checklist
- [x] Tests
- [x] Documentation
- [x] Consolidated commits
- [x] Relevant labels assigned
- [x] Changelog(s) updated

## Steps to Test
1. `yarn test`

## Deploy Notes
None -- deployment is no longer really a thing here.

## Related Issues
Resolves #85 
Related to #66 
Related to #80

### CODE REVIEW

**Comment by slifty** [comment]
NOTE: part of merging this PR is going to be renaming the repository to `countertop`

**Comment by reefdog** [comment]
> Should `CountertopCoordinator` just be called `Countertop` (very important)

**Yes!** I think `CountertopCoordinator` is like saying "we wanted to go with the metaphor, but found the name illegible, so what it really is is a coordinator, so we'll call it both". I'd say call it _either_ the `Countertop` _or_ the `ApplianceCoordinator` (or whatever _thing_/action it is coordinating). IIRC it's doing enough general coordinating that adding a subject could go out of date quickly, so, `Countertop` works as a general name that's clear _enough_ and whose legibility is quickly established once you grok the architecture.

> Is src/classes the right directory (less important)

**I have no opinion!** Semantically it makes sense, but I don't know if there's an idiomatic reason to go with another name!

> Is @tvkitchen/countertop an appropriate package name (last call to change it!).

**Yes!**

File: README.md [hunk 1]
```diff
@@ -1,29 +1,31 @@
...
+# TV Kitchen Countertop
 
 The [TV Kitchen](https://tv.kitchen) is a playground that turns TV into data (transcripts, facial recognition, etc.) so that newsrooms can help their staff and readers better understand what is happening across various local and cable TV news outlets.
 
+The Countertop is the entry point for developers who want to set up a TV Kitchen.
+
 ## Setting up the project
 
-If you are interested in running the TV Kitchen as a developer please review our [install instructions](docs/INSTALL.md).
+If you are interested in running a TV please review our [install instructions](docs/INSTALL.md).
```
Comment (reefdog) [inline README.md:11]: This seems garbled. "running a TV" doesn't seem to be what you mean to say!

File: package.json [hunk 2]
```diff
@@ -25,7 +25,7 @@
     "journalism",
     "news"
   ],
-  "author": "Bad Idea Factory <author@biffud.com>",
+  "author": "Bad Idea Factory <biffuddotcom@biffud.com>",
```
Comment (reefdog) [inline package.json:28]: This email address is a crime, but a delicious crime, like when all that Pappy Van Winkle went missing.

File: README.md (hunk 1 above)
Comment (slifty) [reply README.md]: image

File: package.json (hunk 2 above)
Comment (slifty) [reply package.json:28]: I am genuinely assuming this is [a Wishbone reference](https://www.youtube.com/playlist?list=PLDymFoFsHNt14kLH5F3tGy9x5-579qwUq).

Comment (reefdog) [reply package.json:28]: I just stumbled across this while looking for something else. This is what I get for ignoring all the GitHub notification emails.

No, it was a reference to [this](https://www.huffpost.com/entry/pappy-van-winkle-stolen_n_4114677).

**Pull request merged**